widgets/src/main/resources/org/pentaho/gwt/widgets/themes/public/themes/ruby/globalRuby.css

```diff
@@ -7544,28 +7544,28 @@ span.gwt-RadioButton label {
 
 #filterTypeDeck #staticSelectionTableEditSelected img.image-button-pressed,
 #filterTypeDeck #staticSelectionTableEditSelected img.image-button-over,
-#filterTypeDeck #staticSelectionTableEditSelected img.image-button,
+#filterTypeDeck #staticSelectionTableEditSelected img,
```

Comment:
Are the selectors corresponding to the disabled, pressed and over states, and combinations thereof, still applicable to the new toolbar buttons? Do the new buttons also support these states and need them styles in CSS in another way?

Reply:
selectors corresponding to the disabled and over states are not needed as new buttons are using styles from toolbar-button.